### DISCUSSION TITLE
Documentation - Bad Link

### DISCUSSION THREAD

**Discussion by olddog55** [Motion-Project/motion#1757] [General]
Current documentation at https://motion-project.github.io/motion_config.html#despeckle_filter has a bad or outdated link:

`A very detailed technical explanation of the despeckle part can be found at the webpage of the author of this feature [Ian McConnell's Webcam: Motion Web Page](http://emit.demon.co.uk/motion/)
`

Which unfortunately seems to be off the wire:

````host emit.demon.co.uk 8.8.8.8
Using domain server:
Name: 8.8.8.8
Address: 8.8.8.8#53
Host emit.demon.co.uk not found: 3(NXDOMAIN)
````
Anyone know where this background information on the despeckle filter may have gone to?  A little updating of the web page would seem to be in order.

**Comment by tosiara** [comment]
https://web.archive.org/web/20240000000000*/http://emit.demon.co.uk/motion/

**Comment by olddog55** [comment]
Thank you for that.  As you may know, the Wayback Machine can seem like a big jigsaw at times..

The best links I could find were:
   " https://web.archive.org/web/20190417231544/http://emit.demon.co.uk/motion/  "
For the textual presentation and:
   " https://web.archive.org/web/*/http://emit.demon.co.uk/motion* "
                                                                ^^^ _Note the ' * ' at the URL end..._ ^^^
For a listing of the individual pages to include the raw jpg/png image locations.

If the original Ian McConnell web pages are no longer available, the project documentation should probably be updated to include the Wayback URL's with a note re: the circumstances.
